src/coreclr/jit/instr.cpp

```diff
@@ -2281,6 +2281,8 @@ instruction CodeGen::ins_MathOp(genTreeOps oper, var_types type)
 instruction CodeGen::ins_FloatConv(var_types to, var_types from, emitAttr attr)
 {
     // AVX: For now we support only conversion from Int/Long -> float
+    // AVX512: Supports following conversions
+    //   srcType = ulong                           castToType = double/float
```

Comment:
Could you please fix the formatting of the comment so that both AVX and AVX512 cases have the same style?

Reply:
I have made the suggested changes.